testutil/validatormock/validatormock.go

```diff
@@ -122,7 +132,7 @@ func Attest(ctx context.Context, eth2Cl Eth2Provider, signFunc SignFunc,
 }
 
 // ProposeBlock proposes block for the given slot.
-func ProposeBlock(ctx context.Context, eth2Cl Eth2Provider, signFunc SignFunc, slot eth2p0.Slot) error {
+func ProposeBlock(ctx context.Context, eth2Cl Eth2Provider, signFunc SignFunc, slot eth2p0.Slot, addr string) error {
```

Comment:
add a `TODO: improve validation issue workaround, maybe add the workaround as a wrap of eth2Cl`

Reply:
done